### PR TITLE
Bump grpc to v1.47.0

### PR DESCRIPTION
**Pull request by lucacome**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Bumps grpc to v.1.47.0

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiextensions-apiserver/go.mod [hunk 1]
```diff
@@ -125,6 +125,8 @@ require (
 )
 
 replace (
+	github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.10-0.20210907150352-cf90f659a021
+	github.com/google/go-cmp => github.com/google/go-cmp v0.5.5
```
Comment (aojea) [inline staging/src/k8s.io/apiextensions-apiserver/go.mod:129]: why these replace?
are those added by the scripts?

**Comment by aojea** [comment]
/ok-to-test
let's see what the CI says

File: staging/src/k8s.io/apiextensions-apiserver/go.mod (hunk 1 above)
Comment (lucacome) [reply staging/src/k8s.io/apiextensions-apiserver/go.mod:129]: yeah I just ran the script

**Comment by lucacome** [comment]
Ran 
```
    hack/pin-dependency.sh github.com/envoyproxy/go-control-plane v0.10.2-0.20220325020618-49ff273808a1
    hack/pin-dependency.sh github.com/google/go-cmp v0.5.6
```
as suggested by the linter

**Comment by liggitt** [comment]
needs rebase, then lgtm

**Comment by lucacome** [comment]
> needs rebase, then lgtm

@liggitt thanks! I've rebased.

**Pull request merged**